cl/search/models.py

```diff
@@ -1295,6 +1287,11 @@ class RECAPDocument(AbstractPacerDocument, AbstractPDF, AbstractDateTimeModel):
         ),
         blank=True,
     )
+    acms_document_guid = models.CharField(
+        help_text="The ID of the document in PACER.",
```

Comment:
Err, oops. Maybe we should say "The GUID of the document in ACMS."
Sorry for not catching this earlier.